lib/pbench/server/database/models/datasets.py

```diff
@@ -897,27 +873,29 @@ def get_native_key(key: str) -> str:
 
     @staticmethod
     def is_key_path(key: str, valid: List[str]) -> bool:
-        """
-        Determine whether 'key' is a valid Metadata key path using the list
-        specified in 'valid'. If the "native" key (first element of a dotted
-        path) is in the list, then it's valid.
-
-        NOTE: we only validate the "native" key of the path. The "global"
-        and "user" namespaces are completely open for any subsidiary keys the
-        caller desires. The "dataset" and "server" namespaces are internally
-        defined by Pbench, and can't be modified by the client, however a query
-        for a metadata key that's not defined will simply return None. This
-        seems preferable to building a complicated multi-level keyword path
-        validator and provides a degree of version independence. (That is, if
-        we add "server.nextgenkey" a query for that key on a previous server
-        version will return None rather than failing in validation.)
+        """Determine whether 'key' is a valid Metadata key path using the list
+        specified in 'valid'.
+
+        If the "native" key (first element of a dotted path) is in the list,
+        then it's valid.
+
+        NOTE: we only validate the "native" key of the path.  The "global" and
+        "user" namespaces are completely open for any subsidiary keys the
+        caller desires.  The "dataset" and "server" namespaces are internally
+        defined by Pbench, and can't be modified by the client, however a
+        query for a metadata key that's not defined will simply return None.
+        This seems preferable to building a complicated multi-level keyword
+        path validator and provides a degree of version independence.  (That
+        is, if we add "server.nextgenkey" a query for that key on a previous
+        server version will return None rather than failing in validation.)
 
         Args:
             key: metadata key path
             valid: list of acceptable key paths
 
         Returns:
             True if the path is valid, or False
+
         """
```

Comment:
I don't think this blank line is supposed to be here (e.g., the previous function doesn't have an analogous one...).

Reply:
I'll fix, thanks.

Reply:
Fixed.